Sources/Orbit/Components/Dialog.swift

```diff
@@ -77,21 +78,43 @@ public struct Dialog: View {
 // MARK: - Inits
 extension Dialog {
 
-    /// Creates Orbit Dialog component.
+    /// Creates Orbit Dialog component with a custom content.
     public init(
         illustration: Illustration.Image = .none,
         title: String = "",
         description: String = "",
         style: Style = .primary,
-        alignment: HorizontalAlignment = .leading,
-        buttons: Buttons
+        buttons: Buttons,
+        descriptionLinkAction: @escaping TextLink.Action = { _, _ in },
+        @ViewBuilder content: () -> Content
     ) {
         self.illustration = illustration
         self.title = title
         self.description = description
         self.style = style
-        self.alignment = alignment
         self.buttonConfiguration = buttons
+        self.descriptionLinkAction = descriptionLinkAction
+        self.content = content()
+    }
+
+    /// Creates Orbit Dialog component.
+    public init(
+        illustration: Illustration.Image = .none,
+        title: String = "",
+        description: String = "",
+        style: Style = .primary,
+        buttons: Buttons,
+        descriptionLinkAction: @escaping TextLink.Action = { _, _ in }
+    ) where Content == EmptyView {
```

Comment:
Why not set the default content value in the other init?

```swift
@ViewBuilder content: () -> Content = { EmptyView() }
```

That way, this init wouldn't have to be written.

Reply:
Forgot we can finally do that